api/libproject/devstart.rb

```diff
@@ -742,7 +750,7 @@ def circle_deploy(cmd_name, args)
     exit 1
   end
 
-  if ENV["CIRCLE_BRANCH"] == "master"
+  if ENV.fetch("CIRCLE_BRANCH", "") == "master"
```

Comment:
Maybe save variables above for ENV.fetch("CIRCLE_BRANCH") and/or "isMaster"?

Reply:
done